generate-slr.py

```diff
@@ -69,8 +70,16 @@ def generate_weekly_report(base_url, product, output_dir):
     # TODO: should use pg_slug from PostgreSQL database (but we don't return it right now)
     product_group = report_data['product']['product_group_name'].lower()
 
-    period_from = min(report_data['service_level_objectives'][0]['days'].keys())[:10]
-    period_to = max(report_data['service_level_objectives'][0]['days'].keys())[:10]
+    period_from = period_to = None
+    for slo in report_data['service_level_objectives']:
+        if len(slo['days']):
```

Comment:
To be more pythonic: I would remove the `len(..)` as an empty dict is `False` in Python